SecretListServer/SecretListServer/Auth.swift

```diff
@@ -12,17 +12,49 @@ import Swifter
 private let validEmail = "bas@apple.com"
 private let validPassword = "abcd1234"
 
-func auth(server: HttpServer) {
-  server.post["/auth"] = { r in
-    let fields = r.parseUrlencodedForm().reduce([String: String]()) { map, item in
-      var _map = map
-      _map[item.0] = item.1
-      return _map
+var items = [
+  Item(title: "Buy Milk", createdAt: NSDate()),
+  Item(title: "Sing Karaoke", createdAt: NSDate()),
+  Item(title: "Test drive", createdAt: NSDate()),
+  Item(title: "Visit mom", createdAt: NSDate()),
+  Item(title: "Bake a cake", createdAt: NSDate()),
+]
+
+func allItems(server: HttpServer) {
```

Comment:
These 2 functions are supposed to be in a separate file la.

`Auth.swift`
`Item.swift`